packages/verkle/README.md

```diff
@@ -88,12 +88,13 @@ The `Verkle` class features optional debug logging. Individual debug selections
 
 The following options are available:
 
-| Logger              | Description                                      |
-| ------------------- | ------------------------------------------------ |
-| `verkle`            | minimal info logging for all verkle methods      |
-| `verkle:<METHOD>`   | debug logging for specific verkle method         |
-| `verkle:<METHOD>:*` | verbose debug logging for specific verkle method |
-| `verkle:*`          | verbose debug logging for all verkle methods     |
+| Logger              | Description                          |
+| ------------------- | ------------------------------------ |
+| `verkle:put`        | a verkle put operation has occurred  |
```

Comment:
There should be an addition for `verkle:#` here.

Reply:
Fixed